### PR TITLE
add justinrainbow/json-schema to tooling page

### LINKED ISSUE #944: Add justinrainbow/json-schema to tooling page
### Tool Name

justinrainbow/json-schema

### Tool Description

A PHP Implementation for validating JSON Structures against a given Schema with support for Schemas of Draft-3 or Draft-4

### Action Type

- [X] Add Tool
- [ ] Modify Tool

### Tooling Types

- [X] validator
- [ ] hyper-schema
- [ ] benchmarks
- [ ] documentation
- [ ] LDO-utility
- [ ] code-to-schema
- [ ] data-to-schema
- [ ] model-to-schema
- [ ] schema-to-types
- [ ] schema-to-code
- [ ] schema-to-web-UI
- [ ] schema-to-data
- [ ] util-general-processing
- [ ] util-schema-to-schema
- [ ] util-draft-migration
- [ ] util-format-conversion
- [ ] util-testing
- [ ] editor
- [ ] editor-plugins
- [ ] schema-repository
- [ ] linter
- [ ] linter-plugins

### Languages

PHP

### Environments

PHP >= 7.2

### Dependencies on Validators

_No response_

### Creators

Justin Rainbow - https://github.com/justinrainbow

### Maintainers

Erayd - https://github.com/erayd
David Porter - https://github.com/bighappyface
Jordi Boggiano - https://github.com/Seldaek
Danny van der Sluijs - https://github.com/DannyvdSluijs

### License

MIT

### Source Repository URL

https://github.com/jsonrainbow/json-schema

### Homepage URL

https://github.com/jsonrainbow/json-schema

### Supported Dialects

- [ ] 1
- [ ] 2
- [X] 3
- [X] 4
- [ ] 5
- [ ] 6
- [ ] 7
- [ ] 2019-09
- [ ] 2020-12

### Additional Dialects

_No response_

### Bowtie Compliance Testing

php-justinrainbow-json-schema

### Tooling Listing Notes

_No response_

### Compliance

_No response_

### Landscape Information

logo: https://avatars.githubusercontent.com/u/170307068?s=200&v=4

### PR DESCRIPTION
**Pull request by heysujal**
**What kind of change does this PR introduce?**
This PR adds `justinrainbow/json-schema` to the tooling page as a validator.

As in the mentioned issue, the link for the logo was given. The image is not loading because the URL is not being picked up as it but it is being appended to some directory. Something like below

`http://localhost:3000/img/tools/logos/https:/avatars.githubusercontent.com/u/170307068?s=200&v=4 `

I think this can be fixed by making changes to `next.config.js`
This is another issue on its own and I would like to work on it too. 

**Issue Number:**

-  Closes #944 
-  Similar to  #941 

**Screenshots/videos:**
image

**If relevant, did you update the documentation?**
No

**Summary**

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Review by DarhkVoyd — COMMENTED** [review]
**benjagm** I am unsure about language versioning in `environments`, shouldn't this be in `toolingListingNotes` instead? Also, I believe logos should not be dynamic and be hosted as part of the website.

**Comment by heysujal** [comment]
I have added the image.

File: data/tooling-data.yaml [hunk 1]
```diff
@@ -2409,3 +2409,32 @@
   supportedDialects:
     draft: ['4', '6', '7']
   lastUpdated: '2024-09-03'
+
+- name: justinrainbow/json-schema
+  description: 'A PHP Implementation for validating JSON Structures against a given Schema with support for Schemas of Draft-3 or Draft-4.'
+  toolingTypes: ['validator']
+  languages: ['PHP']
+  environments: ['PHP >= 7.2']
```
Comment (benjagm) [inline data/tooling-data.yaml:2417]: Can you please remove the environment property and provide it in the toolingListingNotes? This is reserved for values like:
                "Web (Online)",
                "GitHub Actions",
                "Command Line",
                "COM/ActiveX"

**Review by benjagm — CHANGES_REQUESTED** [review]
Thanks a lot for your help!!

Can you please take a look to my comment?

File: data/tooling-data.yaml (hunk 1 above)
Comment (heysujal) [reply data/tooling-data.yaml]: I have made the required changes.

**Review by DarhkVoyd — APPROVED** [review]
LGTM!

**Review by DhairyaMajmudar — APPROVED** [review]
Looks Good 🚀

**Review by benjagm — APPROVED** [review]
Looks great! Thanks a lot for this PR.

**Pull request merged**